be/src/olap/rowset/segment_v2/options.h

```diff
@@ -25,6 +25,7 @@ namespace segment_v2 {
 static constexpr size_t DEFAULT_PAGE_SIZE = 1024 * 1024; // default size: 1M
 
 constexpr long ROW_STORE_PAGE_SIZE_DEFAULT_VALUE = 16384; // default row store page size: 16KB
+static constexpr int64_t STORAGE_PAGE_SIZE_DEFAULT_VALUE = 65536;
```

Comment:
use size_t

Reply:
The FE side uses long.
storage_page_size already has a size restriction applied.

Reply:
There are three types for XX_PAGE_SIZE: size_t, long, int64_t. We should use consistent type.

```
static constexpr size_t DEFAULT_PAGE_SIZE = 1024 * 1024; // default size: 1M
constexpr long ROW_STORE_PAGE_SIZE_DEFAULT_VALUE = 16384; // default row store page size: 16KB
static constexpr int64_t STORAGE_PAGE_SIZE_DEFAULT_VALUE = 65536;
```

Reply:
modified